packages/did-provider-oyd/src/oyd-did-provider.ts

```diff
@@ -113,7 +113,7 @@ export class OydDIDProvider extends AbstractIdentifierProvider {
     }
     return context.agent.keyManagerCreate({
       type: args.options.keyType,
-      kms: args.kms || this.defaultKms,
+      kms: args.kms || this.defaultKms || '',
```

Comment:
Rather have a ts-ignore here. Our KMS handles it correctly when no KMS is provided. Using an empty string will result in errors